www/js/app.js

```diff
@@ -514,7 +514,7 @@ define(['jquery', 'zimArchiveLoader', 'uiUtil', 'settingsStore','abstractFilesys
                 // So we have to disable it manually (even if it's still registered and active)
                 navigator.serviceWorker.controller.postMessage({'action': 'disable'});
                 messageChannel = null;
-            }
+              }
```

Comment:
@ram690514  It's minor, but there are two added space here that should be removed.

Reply:
@ram690514 As the only thing missing from this PR was to delete a Tab character, I've done it and pushed a commit.